src/classes/SRVC_ContactPrimaryLanguage_TEST.cls

```diff
@@ -0,0 +1,637 @@
+        Test.startTest();
+        List<Contact_Language__c> conLangsToSetNonPrimary = SRVC_ContactPrimaryLanguage.getInstance().getContactLanguagesToUpdateAsNonPrimary(new Set<Id>());
+        Test.stopTest();
+
+        System.assertEquals(true, conLangsToSetNonPrimary.isEmpty(), 'There should be no Contact Languages to be updated to non-primary status.');
+    }
+
+    /**************************************************************************************************************************
+    * @description Test method to verify that the getDefaultFluencyFromSettings correctly returns the value set as the 
+    * Default_Contact_Language_Fluency__c.
+    ***************************************************************************************************************************/
+    @isTest 
+    private static void getDefaultFluencyFromSettingsPositive(){
+        Hierarchy_Settings__c hs = UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c
+                                                                                    (Default_Contact_Language_Fluency__c = 'Fluent'));
+
+        Test.startTest();
+        String fluencySetting = SRVC_ContactPrimaryLanguage.getInstance().getDefaultFluencyFromSettings();
+        Test.stopTest();
+
+        System.assertEquals('Fluent', fluencySetting, 'Default fluency setting should be "Fluent".');
+    }
+
+    /**************************************************************************************************************************
+    * @description Test method to verify that the getDefaultFluencyFromSettings correctly returns a blank value when 
+    * the value of Default_Contact_Language_Fluency__c is set as null.
+    ***************************************************************************************************************************/
+    @isTest 
+    private static void getDefaultFluencyFromSettingsNull(){
+        Hierarchy_Settings__c hs = UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c
+                                                                                     (Default_Contact_Language_Fluency__c = null));
+
+        Test.startTest();
+        String fluencySetting = SRVC_ContactPrimaryLanguage.getInstance().getDefaultFluencyFromSettings();
+        Test.stopTest();
+
+        System.assertEquals(true, String.isBlank(fluencySetting), 'Default fluency setting should be null.');
+    }
+
+    /**************************************************************************************************************************
+    * @description Test method to verify that the getDefaultFluencyFromSettings correctly returns a blank value when 
+    * the value of Default_Contact_Language_Fluency__c is blank.
+    ***************************************************************************************************************************/
+    @isTest 
+    private static void getDefaultFluencyFromSettingsBlank(){
```

Comment:
@m-kinkade What value does this test add to our suite in light of the other tests?